rye/src/cli/fetch.rs

```diff
@@ -10,7 +10,7 @@ use crate::pyproject::PyProject;
 use crate::sources::py::PythonVersionRequest;
 use crate::utils::CommandOutput;
 
-/// Fetches a Python interpreter for the local machine. This is an alias of `rye toolchain fetch`.
+/// Fetches a Python interpreter for the local machine
```

Comment:
This note is also visible in `rye -h` for fetch. Does that change too with this PR, or does it come from somewhere else?